swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
```

Comment:
For keyword arguments, [don't surround your `=` with whitespace](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements:~:text=Don%E2%80%99t%20use%20spaces%20around%20the%20%3D%20sign%20when%20used%20to%20indicate%20a%20keyword%20argument%2C%20or%20when%20used%20to%20indicate%20a%20default%20value%20for%20an%20unannotated%20function%20parameter%3A)
```suggestion
    def __init__(self, inventory=None):
```